HARK/ConsumptionSaving/tests/test_IndShockConsumerType.py

```diff
@@ -698,9 +701,12 @@ def test_Harmenberg_mtd(self):
         c_std1 = np.std(Consumption_list)
         c_std_ratio = c_std2 / c_std1
 
-        self.assertAlmostEqual(c_std2, 0.03768819564871894)
-        self.assertAlmostEqual(c_std1, 0.004411745897568616)
-        self.assertAlmostEqual(c_std_ratio, 8.542694099741672)
+        # simulation tests -- seed/generator specific
+        # But these are based on aggregate population statistics.
+        # WARNING: May fail stochastically, or based on specific RNG types.
+        self.assertAlmostEqual(c_std2, 0.0376882)
```

Comment:
If we keep these in we might want a low tolerance for them?

Reply:
that's a good idea.

Reply:
But if there's no objection to me removing them, and it sounds like there isn't, I will.

I'll respond to your point about tests of sampled moments in the main thread.